virttest/utils_libvirt/libvirt_network.py

```diff
@@ -104,3 +105,60 @@ def check_established(params):
             raise exceptions.TestFail("Pattern '%s' is not matched in "
                                       "'%s'" % (pat_str,
                                                 cmdRes.stdout_text.strip()))
+
+
+def modify_network_xml(net_dict, testnet_xml):
+    """
+    modify the network's xml
+    :param testnet_xml: the network xml object to be modified
+    :param net_dict: The dict restore need updated items like mac, bandwidth, forward
+    :return: the updated network xml
+    """
+    del_nat = net_dict.get('del_nat')
+    del_ip = net_dict.get('del_ip')
+    dns_txt = net_dict.get('dns_txt')
+    domain = net_dict.get('domain')
+    bridge = net_dict.get('bridge')
+    forward = net_dict.get('forward')
+    # get the params about forward interface
+    interface_dev = net_dict.get('interface_dev')
+    virtualport_type = net_dict.get('virtualport')
+    # bandwidth to be set if any
+    net_bandwidth_outbound = net_dict.get('net_bandwidth_outbound')
+    net_bandwidth_inbound = net_dict.get('net_bandwidth_inbound')
+    mac = net_dict.get("mac")
+
+    # delete the <bridge/> and <mac/> elements as they can be
+    # generated automatically if needed
+    testnet_xml.del_bridge()
+    testnet_xml.del_mac()
+    if del_nat:
+        testnet_xml.del_nat_attrs()
+    if del_ip:
```

Comment:
It would be much safer if we use `if del_nat is True`.
When we're calling this function, del_nat could be assigned with other type of value and we don't want to confuse the function.

Reply:
Updated, Thank you!